src/pykeen/lr_schedulers/__init__.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+
+"""Learning Rate Schedulers available in PyKEEN.
+
+===========================  =============================================================
+Name                         Reference
+===========================  =============================================================
+CosineAnnealingLR            :class:`torch.optim.lr_scheduler.CosineAnnealingLR`
+CosineAnnealingWarmRestarts  :class:`torch.optim.lr_scheduler.CosineAnnealingWarmRestarts`
+CyclicLR                     :class:`torch.optim.lr_scheduler.CyclicLR`
+ExponentialLR                :class:`torch.optim.lr_scheduler.ExponentialLR`
+LambdaLR                     :class:`torch.optim.lr_scheduler.LambdaLR`
+MultiplicativeLR             :class:`torch.optim.lr_scheduler.MultiplicativeLR`
+MultiStepLR                  :class:`torch.optim.lr_scheduler.MultiStepLR`
+OneCycleLR                   :class:`torch.optim.lr_scheduler.OneCycleLR`
+StepLR                       :class:`torch.optim.lr_scheduler.StepLR`
+===========================  =============================================================
+
+.. note:: This table can be re-generated with ``pykeen ls lr_schedulers -f rst``
+"""
+
+from typing import Any, Mapping, Type
+
+from class_resolver import Resolver
+from torch.optim.lr_scheduler import (
+    _LRScheduler,
+    CosineAnnealingLR,
+    CosineAnnealingWarmRestarts,
+    CyclicLR,
+    ExponentialLR,
+    LambdaLR,
+    MultiplicativeLR,
+    MultiStepLR,
+    OneCycleLR,
+    StepLR,
+)
+
+__all__ = [
+    '_LRScheduler',
+    'lr_schedulers_hpo_defaults',
+    'lr_scheduler_resolver',
+]
```

Comment:
we could dynamically add all subclasses of _LRScheduler to the `__all__` so they get autodoc'd

Reply:
Will you take care of this?

Reply:
Yes

Reply:
Done in a15562f0